pom.xml

```diff
@@ -594,6 +594,32 @@
                 </executions>
             </plugin>
 
+            <!-- Scala style check -->
+            <plugin>
+                <groupId>org.scalastyle</groupId>
+                <artifactId>scalastyle-maven-plugin</artifactId>
+                <version>0.8.0</version>
+                <configuration>
+                    <verbose>false</verbose>
+                    <failOnViolation>false</failOnViolation>
+                    <failOnWarning>false</failOnWarning>
+                    <includeTestSourceDirectory>true</includeTestSourceDirectory>
+                    <sourceDirectory>${project.basedir}/src/main/scala</sourceDirectory>
+                    <testSourceDirectory>${project.basedir}/src/test/scala</testSourceDirectory>
+                    <detail>true</detail>
+                    <configLocation>${user.dir}/scalastyle_config.xml</configLocation>
```

Comment:
I guess the real solution would be to use the [directory-maven-plugin](https://github.com/jdcasey/directory-maven-plugin) or define a property `root.dir` that is set in every submodule. But since we mostly care about it being checked by Travis, `user.dir` is fine.